Building.md

```diff
@@ -5,7 +5,7 @@ Documentation][cuda_quantum_docs]. The page also contains [installation
 instructions][official_install] for released packages and nightly builds.
 
 [cuda_quantum_docs]: https://nvidia.github.io/cuda-quantum/latest
-[official_install]: https://nvidia.github.io/cuda-quantum/latest/install.html
+[official_install]: https://nvidia.github.io/cuda-quantum/latest/using/quick_start.html#install-cuda-q
 [data_center_install]:
     https://nvidia.github.io/cuda-quantum/latest/data_center_install.html
```

Comment:
As long as we've got the hood open, I think it would be worthwhile to fix the data_center_install link, too. I think there's a similar change required in Dev_Setup.md. Would you mind making the change there as well?

```suggestion
[official_install]: https://nvidia.github.io/cuda-quantum/latest/using/quick_start.html#install-cuda-q
[data_center_install]:
    https://nvidia.github.io/cuda-quantum/latest/using/install/data_center_install.html
```

Reply:
Thank you for your comment. I updated!